src/backend/access/transam/xlog.c

```diff
@@ -8207,7 +8211,8 @@ StartupXLOG(void)
 	if (fast_promoted)
 		RequestCheckpoint(CHECKPOINT_FORCE);
 
-	*shmCleanupBackends = true;
+	 if (Gp_role == GP_ROLE_DISPATCH)
```

Comment:
Remove the extra space.
```suggestion
	if (Gp_role == GP_ROLE_DISPATCH)
```